src/action/common/provision_nix.rs

```diff
@@ -65,14 +88,21 @@ impl Action for ProvisionNix {
     fn execute_description(&self) -> Vec<ActionDescription> {
         let Self {
             fetch_nix,
-            create_users_and_group,
+            delete_users,
+            create_group,
             create_nix_tree,
             move_unpacked_nix,
         } = &self;
 
         let mut buf = Vec::default();
         buf.append(&mut fetch_nix.describe_execute());
-        buf.append(&mut create_users_and_group.describe_execute());
+
+        // TODO: This is a bit... loud
```

Comment:
Is there an obvious way in which we can improve it? If not, I'd be inclined to just drop the TODO and revisit if someone complains.

Reply:
We could make a meta-action that encapsulates these and has a shorter description.... I'm trying to weigh how I feel about "communicating explicitly and obviously to the user that we will be removing users that are cruft" and "providing a short plan output".

I think ideally we'd output "Delete 32 users inside the group nixbld because we don't need them anymore" and have that in like, yellow or red to draw attention to the fact that is it a cure step.

Reply:
Alright, let's keep this around until we settle on one or the other, then!